src/test/kotlin/lotto/LottoCalculatorTest.kt

```diff
@@ -0,0 +1,15 @@
+package lotto
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+
+class LottoCalculatorTest {
```

Comment:
```suggestion
class CalculatorTest {
```
대체로 테스트 코드의 클래스명은 `클래스명Test` 로 작성합니다. 😅